### PR TITLE
updated endpoint and fixed button status

### PR DESCRIPTION
**Pull request by danymill**

### CODE REVIEW

**Review by dzmitrymishyn — APPROVED** [review]

File: client/app/containers/test-details/test-details.controller.js [hunk 1]
```diff
@@ -372,12 +377,47 @@ const testDetailsController = function testDetailsController(
...
+        const params = tests.map(test => {
+            const testEvent = createWorkItem('EVENT', test, message);
+
+            return {
+                testId: test.id,
+                workItems: [testEvent]
+            };
+        });
+
+        console.log(params);
```
Comment (DmitryMishin) [inline client/app/containers/test-details/test-details.controller.js:402]: do you need that?

File: client/app/containers/test-details/test-details.controller.js [hunk 2]
```diff
@@ -372,12 +377,47 @@ const testDetailsController = function testDetailsController(
...
+            });
+    }
+
+    function bulkCreateWorkItems(message, tests) {
+        const params = tests.map(test => {
+            const testEvent = createWorkItem('EVENT', test, message);
+
+            return {
+                testId: test.id,
+                workItems: [testEvent]
```
Comment (DmitryMishin) [inline client/app/containers/test-details/test-details.controller.js:398]: add comma

**Review by dzmitrymishyn — APPROVED** [review]

File: client/app/containers/test-details/test-details.controller.js (hunk 1 above)
Comment (danymill) [reply client/app/containers/test-details/test-details.controller.js]: fixed

File: client/app/containers/test-details/test-details.controller.js (hunk 2 above)
Comment (danymill) [reply client/app/containers/test-details/test-details.controller.js]: fixed

**Pull request merged**